test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NetCoreProjectTestCase.cs

```diff
@@ -258,5 +258,100 @@ public static IEnumerable<object[]> GetNetCoreTemplates()
         {
             yield return new object[] { ProjectTemplate.NetStandardClassLib };
         }
+
+        [NuGetWpfTheory]
+        [MemberData(nameof(GetNetCoreProjects))]
+        public void InstallPackageToNetCoreProjectFromUI(ProjectTemplate projectTemplate, string projectName, string packageName, string packageVersion)
```

Comment:
Since there's only one test case defined in `GetNetCoreProjects`, how about making it a `NuGetWpfFact`, and using the string literals in the test code below?  That way when reading the test code, I don't need to jump around to different parts of the file to see what values are actually being passed in.

Reply:
Hi @zivkan, actually we have 5 test cases defined in GetNetCoreProjects as below (changed the last package to "Newtonsoft.Json" just now) when we were writing the series of test cases. However, we just kept one because we worry about the time problem if adding them all to everyday build. So, should we try the five cases with [NuGetWpfTheory] as the original idea or change them to [NuGetWpfFact]?

```
        public static IEnumerable<object[]> GetNetCoreProjects()
        {
            yield return new object[] { ProjectTemplate.NetCoreConsoleApp, "NetCoreConsole", "Automapper", "11.0.0" };
            yield return new object[] { ProjectTemplate.NetCoreClassLib, "NetCoreClassLib", "Microsoft.NETCore.Platforms", "6.0.6" };
            yield return new object[] { ProjectTemplate.NetCoreXUnitTest, "NetCoreXUnit", "NUnit", "3.13.2" };
            yield return new object[] { ProjectTemplate.ASPNETCoreWebApplication, "NetCoreWeb", "Microsoft.NETCore.Targets", "3.1.0" };
            yield return new object[] { ProjectTemplate.NetStandardClassLib, "NetStandardClassLib", "Newtonsoft.Json", "13.0.1" };
        }
```

Reply:
You must be looking at something different to me, because in this PR I see:

* [`GetNetCoreProjects`](https://github.com/NuGet/NuGet.Client/pull/4999/files#diff-15febf9b9c2640538162ca786717f618186c3c0b448c4d41dde8537354b23e23R287-R290)
* [`UpdateNetCoreProjects`](https://github.com/NuGet/NuGet.Client/pull/4999/files#diff-15febf9b9c2640538162ca786717f618186c3c0b448c4d41dde8537354b23e23R320-R323)
* [`UninstallNetCoreProjects`](https://github.com/NuGet/NuGet.Client/pull/4999/files#diff-15febf9b9c2640538162ca786717f618186c3c0b448c4d41dde8537354b23e23R352-R355)

Each of which only have 1 case.

Reply:
Sorry, I didn't describe clearly. At the beginning, we wanted to define 5 test cases as above. But we thought it would waste time for our Dev build if merge all the 5 ones. So, we deleted 4 from it and reserved the fifth project "NetStandardClassLib" as you see in the PR.